src/test/scala/nl.knaw.dans.easy.properties/server/GraphQLResolveSpec.scala

```diff
@@ -1553,6 +1559,11 @@ class GraphQLResolveSpec extends TestSupportFixture
     runQuery(input)
   }
 
+  private def inAnyOrder[T](expected: Seq[T]): FunctionAdapter1[Seq[T], Boolean] = {
+    // TODO vararch? To some package or find in library.
```

Comment:
or find out why it doesn't require the `inAnyOrder` on my machine nor on Travis, but why it does require it on your end 😉

Reply:
The tests run with blessed/master, so its caused by my changes, or problems that still have to be fixed.

Reply:
If it's caused by your change, let's for now just revert this `inAnyOrder`, put the elements in the order that the test expects them to be in and put a TODO somewhere to indicate that we need to take a good look at it. To be completely honest with you, I remember having to deal with this issue before while writing these tests and as you may have seen, some of them already use this proposed 'quick fix'. I've had the same problem and this is how I solved it. I'm not particularly worried about this ordering thing, especially given that we're talking about mocks here.